Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Data/Security/1030-TPDM-DeletePerformanceEvaluationRatingsAuthorizationStrategy.sql

```diff
@@ -0,0 +1,23 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DECLARE 
+	@claimId AS INT,
+    @claimName AS nvarchar(max)
+
+SET @claimName = 'http://ed-fi.org/ods/identity/claims/tpdm/performanceEvaluationRating'
+
+SELECT @claimId = ResourceClaimId
```

Comment:
DECLARE 
	@claimId AS INT,
    @claimName AS nvarchar(max),
    @parentResourceClaimId AS nvarchar(max)

SET @claimName = 'http://ed-fi.org/ods/identity/claims/tpdm/performanceEvaluationRating'

SELECT @parentResourceClaimId= ParentResourceClaimId,@claimId= ResourceClaimId
FROM dbo.ResourceClaims WHERE claimName ='http://ed-fi.org/ods/identity/claims/tpdm/performanceEvaluationRating'


IF EXISTS ( SELECT 1 FROM dbo.ResourceClaims  where ResourceClaimId = @parentResourceClaimId)
BEGIN

        IF EXISTS ( SELECT 1 FROM dbo.ResourceClaims rc
                    INNER JOIN dbo.ResourceClaimActions rca on rca.ResourceClaimId = rc.ResourceClaimId
                    INNER JOIN dbo.ResourceClaimActionAuthorizationStrategies rcaas on rcaas.ResourceClaimActionId = rca.ResourceClaimActionId
                    WHERE rc.ResourceClaimId = @parentResourceClaimId)
        BEGIN
                    -- Setting default authorization metadata
                     IF EXISTS (SELECT 1
                                    FROM dbo.ResourceClaims rc
                                    INNER JOIN dbo.ResourceClaimActions rca on rca.ResourceClaimId = rc.ResourceClaimId
                                    INNER JOIN dbo.ResourceClaimActionAuthorizationStrategies rcaas on rcaas.ResourceClaimActionId = rca.ResourceClaimActionId
                                    WHERE rc.ResourceClaimId = @claimId )
                     BEGIN
                                    PRINT 'Deleting default action authorizations for resource claim ''' + @claimName + ''' (claimId=' + CONVERT(nvarchar, @claimId) + ').'
                                    DELETE rcaas
                                    FROM dbo.ResourceClaims rc
                                    INNER JOIN dbo.ResourceClaimActions rca on rca.ResourceClaimId = rc.ResourceClaimId
                                    INNER JOIN dbo.ResourceClaimActionAuthorizationStrategies rcaas on rcaas.ResourceClaimActionId = rca.ResourceClaimActionId
                                    WHERE rc.ResourceClaimId = @claimId
       END          END      

END
GO